Library/Formula/aws-elasticache.rb

```diff
@@ -1,14 +1,18 @@
-require 'formula'
-
 class AwsElasticache < AmazonWebServicesFormula
-  homepage 'http://aws.amazon.com/developertools/2310261897259567'
-  url 'https://s3.amazonaws.com/elasticache-downloads/AmazonElastiCacheCli-2013-06-15-1.9.000.zip'
-  version '1.9.000'
-  sha1 'c8962cc2ea22f88883b38cda2674d1befbed1b71'
+  homepage "https://aws.amazon.com/developertools/2310261897259567"
+  url "https://s3.amazonaws.com/elasticache-downloads/AmazonElastiCacheCli-latest.zip"
```

Comment:
We don't like using `-latest` links as they break on new versions. Can you try and find a versioned one instead? Thanks!